### PR TITLE
Implemented sockets.io

### PR DESCRIPTION
**Pull request by garzeah**
**Updates**
- Real-time communication works now!

Just need to implement the Google Translate API and work on mobile view! :D

### CODE REVIEW

**Comment by bonnieli** [comment]
hey @garzeah please change these bases off of which branch you based it off of so it reflects the proper file changes

**Comment by garzeah** [comment]
> hey @garzeah please change these bases off of which branch you based it off of so it reflects the proper file changes

Just did it!

File: client/src/pages/Messenger.js [hunk 1]
```diff
@@ -1,33 +1,26 @@
...
+	SOCKET_SERVER_URL = socketServerURL();
 
 const Messenger = () => {
 	// Your profile information
 	const [user, setUser] = useState({});
 	// Keeps track of your current conversation
 	const [currConvo, setCurrConvo] = useState(null);
-	const socketRef = useRef();
+	// Keeps track of whether message is sent and triggers re-render
+	const [isMessageSent, setIsMessageSent] = useState({ status: false });
```
Comment (bonnieli) [inline client/src/pages/Messenger.js]: will you be extending this more in the future? if not, instead of an object, just store a boolean

Comment (garzeah) [reply client/src/pages/Messenger.js]: Nope, not at this moment. I just stored it as boolean!

**Review by bonnieli — APPROVED** [review]

**Pull request merged**